schema.sql

```diff
@@ -83,4 +83,13 @@ CREATE TABLE visits (
     FOREIGN KEY (animal_id) REFERENCES animals (id) ON DELETE CASCADE,
     FOREIGN KEY (vets_id) REFERENCES vets (id) ON DELETE CASCADE,
```

Comment:
Please add a project title and description in the README file. A professional README file should have a project title and project description.